### DISCUSSION TITLE
Automatic lock of the poll

### DISCUSSION THREAD

**Discussion by sramekmichal** [lukevella/rallly#441] [Ideas]
Sometimes it is good to have automatic poll lock.

As an example, you need a list of attendees to buy food for them. For that particular reason, you would set up automatic lock 2 days before (at 00:00), which would work as a deadline for participants :-)

**Comment by lukevella** [comment]
Hi @sramekmichal, please use discussions for feature requests. Trying to keep the issues tab clean to see actionable items. Moving this to discussions.

**Reply by sramekmichal** [reply]
Sorry, I am stupid to not notice it...

**Reply by lukevella** [reply]
No problem. I can confirm that setting deadlines would be great and that's certainly on the roadmap 👍
